applications/openshift/logging/file_permissions_var_log_kube_audit/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Kubernetes Audit Logs Must Have Mode 0600'
+
+platforms:
+  - ocp4-master-node
+
+description: |-
+    {{{ describe_file_permissions(file="/var/log/kube-apiserver/.*", perms="0600") }}}
+
+rationale: 'If users can write to audit logs, audit trails can be modified or destroyed.'
+
+severity: medium
+
+identifiers:
+    cce@ocp4: CCE-83654-4
+
+references:
+    cjis: 5.4.1.1
+    cui: 3.3.1
+    nist: CM-6(a),AC-6(1),AU-9(4)
+    nist-csf: DE.AE-3,DE.AE-5,PR.AC-4,PR.DS-5,PR.PT-1,RS.AN-1,RS.AN-4
+    pcidss: Req-10.5
+    isa-62443-2013: 'SR 2.1,SR 2.10,SR 2.11,SR 2.12,SR 2.8,SR 2.9,SR 5.2,SR 6.1'
+    isa-62443-2009: 4.2.3.10,4.3.3.3.9,4.3.3.5.8,4.3.3.7.3,4.3.4.4.7,4.3.4.5.6,4.3.4.5.7,4.3.4.5.8,4.4.2.1,4.4.2.2,4.4.2.4
+    cobit5: APO01.06,APO11.04,APO12.06,BAI03.05,BAI08.02,DSS02.02,DSS02.04,DSS02.07,DSS03.01,DSS05.04,DSS05.07,DSS06.02,MEA02.01
+    iso27001-2013: A.10.1.1,A.11.1.4,A.11.1.5,A.11.2.1,A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1,A.13.1.1,A.13.1.3,A.13.2.1,A.13.2.3,A.13.2.4,A.14.1.2,A.14.1.3,A.16.1.4,A.16.1.5,A.16.1.7,A.6.1.2,A.7.1.1,A.7.1.2,A.7.3.1,A.8.2.2,A.8.2.3,A.9.1.1,A.9.1.2,A.9.2.3,A.9.4.1,A.9.4.4,A.9.4.5
+    cis-csc: 1,11,12,13,14,15,16,18,19,3,4,5,6,7,8
+
+ocil_clause: '{{{ ocil_clause_file_permissions(file="/var/log/kube-apiserver/.*", perms="-rwx------") }}}'
```

Comment:
the template says `0600` but the `ocil` and `ocil_clause` both say `0700`

Reply:
oops! Let me fix it

Reply:
fixed.